components/atom/input/src/Addon/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomInput-addon'
+
+const TYPES = {
+  LEFT: 'left',
+  RIGHT: 'right'
+}
+
+const getClassName = ({type}) =>
+  cx(
+    BASE_CLASS,
+    `${BASE_CLASS}-${type}`
+  )
+
+const Addon = ({type, label}) =>
+  <span className={getClassName({type})}>
+    {label}
+  </span>
+
+Addon.displayName = 'Addon'
+
+Addon.propTypes = {
+  /**
+   * Text to be shown within the addon
+   */
+  label: PropTypes.string,
+  /**
+   * 'left' or 'right'
+   */
+  type: PropTypes.oneOf(Object.values(TYPES)),
+}
+
+export default Addon
```

Comment:
sorry, i don't understand the concept of "addon". what's the purpose of it? it's just for alignment?

Reply:
These are addons

![image](https://user-images.githubusercontent.com/17549662/39746534-dee8c6c6-52aa-11e8-9659-97b3b11020cb.png)

Reply:
ok, thank you!